addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1347,6 +1349,15 @@ var Wysiwyg = Widget.extend({
         }
         this.$attribute_translations = $('.o_editable_translatable_attribute');
         this.translations = [];
+        // (id -> value) map of the initial values of a translation, to be able to determine
+        // whether a translation has been modified or not, so as to not save unmodified translation
+        // (important because when a translation is saved, Odoo marks it as translated)
+        const translationsInitialValues = $('[data-oe-translation-id]').toArray().reduce((acc, elem) => {
+            acc[elem.getAttribute('data-oe-translation-id')] = elem.textContent
```

Comment:
1) Did you check that it's only text that is saved? Does it work for image "alt" translation as well?
2) missing semicollon;